src/Microsoft.Azure.SignalR.Serverless.Protocols/JsonServerlessProtocol.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.IO;
+using System.Text;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols
+{
+    public class JsonServerlessProtocol : IServerlessProtocol
```

Comment:
this sounds more like a parser than a protocol?

Reply:
But `TryParseMessage` is also a functionality in SignalR protocol. It just miss the `GetMessageBytes` here.